supertokens_python/recipe/session/utils.py

```diff
@@ -288,6 +291,31 @@ async def default_invalid_claim_callback(
     )
 
 
+def get_auth_mode_from_header(request: BaseRequest) -> Optional[str]:
```

Comment:
This is defined twice (once here once in `cookie_and_header`)

Reply:
Removed from `cookie_and_header.py`